runtime/syntax/hgcommitDiff.vim

```diff
@@ -0,0 +1,391 @@
+" Vim syntax file
+" Language:	Sapling / Mecurial Diff (context or unified)
+" Maintainer:	Bram Moolenaar <Bram@vim.org>
+"               Max Coplan <mchcopl@gmail.com>
+"               Translations by Jakson Alves de Aquino.
+" Last Change:	2022-12-08
+" Copied from:	runtime/syntax/diff.vim
+
+" Quit when a (custom) syntax file was already loaded
+if exists("b:current_syntax")
+  finish
+endif
+scriptencoding utf-8
+
+syn match diffOnly	"^\%(SL\|HG\): Only in .*"
```

Comment:
Isn't it better to use the different group name(s)?
```suggestion
syn match hgdiffOnly	"^\%(SL\|HG\): Only in .*"
```

Reply:
yes.  Will fix

Reply:
done